contrib/win32/win32compat/termio.c

```diff
@@ -247,6 +245,19 @@ syncio_initiate_write(struct w32_io* pio, DWORD num_bytes)
 	return 0;
 }
 
+static BOOL
+is_windows8_or_Later()
+{
+	OSVERSIONINFO osvi;
+
+	ZeroMemory(&osvi, sizeof(OSVERSIONINFO));
+	osvi.dwOSVersionInfoSize = sizeof(OSVERSIONINFO);
+
+	GetVersionEx(&osvi);
```

Comment:
As per MSDN, 
[GetVersionEx may be altered or unavailable for releases after Windows 8.1. Instead, use the Version Helper functions]

we have a simpler function IsWindows8OrGreater()
https://msdn.microsoft.com/en-us/library/windows/desktop/dn424961(v=vs.85).aspx

Reply:
changed to use  IsWindows8OrGreater()